src/twisted/web/test/test_http_headers.py

```diff
@@ -611,14 +610,14 @@ def test_subclassRepr(self):
         The L{repr} of an instance of a subclass of L{Headers} uses the name
         of the subclass instead of the string C{"Headers"}.
         """
-        foo = u"foo\u00E1"
-        bar = u"bar\u2603"
-        baz = u"baz"
+        foo = "foo\u00E1"
+        bar = "bar\u2603"
+        baz = "baz"
         fooEncoded = "'foo\\xe1'"
         barEncoded = "'bar\\xe2\\x98\\x83'"
-        if _PY3:
-            fooEncoded = "b" + fooEncoded
-            barEncoded = "b" + barEncoded
+        fooEncoded = "b" + fooEncoded
+        barEncoded = "b" + barEncoded
```

Comment:
Remove the 2 lines above and define them directly as
```suggestion
        fooEncoded = "b'foo\\xe1'"
        barEncoded = "b'bar\\xe2\\x98\\x83'"
```

Reply:
OK, fixed.